README.md

```diff
@@ -58,6 +58,25 @@ You can run the tests with `python -m pytest` once your environemnt is configure
 All tests are run automatically by CircleCI when you create a PR that connects to master.  Please make
 sure tests run relatively quickly (seconds, not minutes).
 
+## Authentication
+
+Endpoints that create/update data are authenticated with a JWT bearer token. 
+
+To obtain a token, run:
+```shell
+flask auth getToken tokenName
+```
+
+The token is secured by the value of `FLASK_SECRET`, so tokens generated in one environment will not work elsewhere 
```

Comment:
do you mean `SECRET_KEY`?

Reply:
Yes, but think how much fun I could have had spending an hour swearing at it when I used the wrong environment variable name on the server and couldn't figure out what was wrong